content/swagger/resources/measurement_families/routes/measurement_families.yaml

```diff
@@ -0,0 +1,165 @@
+                  },{
+                    "code": "SQUARE_DECIMETER",
+                    "labels": {
+                      "en_US": "Square decimeter",
+                      "fr_FR": "Décimètre carré"
+                    },
+                    "convert_from_standard": [
+                      {
+                        "operator": "mul",
+                        "value": "0.000001"
+                      }
+                    ],
+                    "symbol": "dm\u00b2"
+                  },{
+                    "code": "SQUARE_METER",
+                    "labels": {
+                      "en_US": "Square meter",
+                      "fr_FR": "Mètre carré"
+                    },
+                    "convert_from_standard": [
+                      {
+                        "operator": "mul",
+                        "value": "1"
+                      }
+                    ],
+                    "symbol": "m\u00b2"
+                  },{
+                    "code": "CENTIARE",
+                    "labels": {
+                      "en_US": "Centiare",
+                      "fr_FR": "Centiare"
+                    },
+                    "convert_from_standard": [
+                      {
+                        "operator": "mul",
+                        "value": "1"
+                      }
+                    ],
+                    "symbol": "ca"
+                  },{
+                    "code": "SQUARE_DEKAMETER",
+                    "labels": {
+                      "en_US": "Square dekameter",
+                      "fr_FR": "Décamètre carré"
+                    },
+                    "convert_from_standard": [
+                      {
+                        "operator": "mul",
+                        "value": "100"
+                      }
+                    ],
+                    "symbol": "dam\u00b2"
+                  },{
+                    "code": "ARE",
+                    "labels": {
+                      "en_US": "Are",
+                      "fr_FR": "Are"
+                    },
+                    "convert_from_standard": [
+                      {
+                        "operator": "mul",
+                        "value": "1"
+                      }
+                    ],
+                    "symbol": "a"
+                  },{
+                    "code": "SQUARE_HECTOMETER",
+                    "labels": {
+                      "en_US": "Square hectometer",
+                      "fr_FR": "Hectomètre carré"
+                    },
+                    "convert_from_standard": [
+                      {
+                        "operator": "mul",
+                        "value": "10000"
+                      }
+                    ],
+                    "symbol": "hm\u00b2"
+                  }
+              ]
+            }
+          ]
+        }
+      }
+    401:
+      $ref: "#/responses/401Error"
+    403:
+      $ref: "#/responses/403Error"
```

Comment:
```suggestion
```
This error can only appear if the API user doesn't have the right to see this endpoint. But as there is no ACL implemented for this endpoint, it cannot happen.

Reply:
Merci pour l'info :)